### PR TITLE
Drop now optional requirements...

### PR DESCRIPTION
**Pull request by dwbrown2**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,9 +16,7 @@ Here is a summary of features enabled by this cost model:
 ## Requirements
 
 - Kubernetes version 1.8 or higher
-- kube-state-metrics
-- Node exporter
```
Comment (michaelmdresser) [inline README.md:20]: This is true in the context of exporter-only, right? But not regular installs?

Comment (dwbrown2) [reply README.md:20]: So node-exporter is 100% not required in any mode.

I believe that ksm is "optional" but it does give you extra metrics, e.g. PVs. So maybe we drop node-exporter and caveat ksm?

Comment (dwbrown2) [reply README.md:20]: cc @AjayTripathy just in case he has a different perspective

Comment (michaelmdresser) [reply README.md:20]: > So maybe we drop node-exporter and caveat ksm?

I agree with this idea.

**Review by michaelmdresser — APPROVED** [review]

**Pull request merged**